app/assets/stylesheets/footer.scss

```diff
@@ -32,7 +32,7 @@
 }
 
 .bounce-out-on-hover {
-  box-shadow: 0 0 1px $black;
```

Comment:
I think if not needed, we can remove the padding that looks a little large between item

Reply:
I didn't get it, whether there is any issue with removing the box shadow? Ref https://github.com/CircuitVerse/CircuitVerse/pull/2591

Reply:
Checked, I find no border much better UX